qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -144,10 +144,10 @@ def __init__(
         azimuthal_angles = np.asarray(azimuthal_angles, dtype=float)
         detunings = np.asarray(detunings, dtype=float)
 
-        # check if all the rabi_rates are greater than zero
+        # check if all the rabi_rates are non-negative
         check_arguments(
             all(rabi_rates >= 0.0),
-            "All Rabi rates must be greater than zero.",
+            "All Rabi rates must be non-negative.",
```

Comment:
Should we instead use nonnegative?
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/n/non
Or is this a case when the hyphen avoids confusion?

Reply:
Although maybe since we've always used non-negative for error messages we should just use that here and perhaps change it everywhere at a later date.

Reply:
Followed the design doc on this, but should be without a hyphen.